package.json

```diff
@@ -13,5 +13,12 @@
   "volta": {
     "node": "18.16.0",
     "pnpm": "8.6.2"
+  },
+  "pnpm": {
+    "peerDependencyRules": {
+      "ignoreMissing": [
+        "webpack"
```

Comment:
added because `ember-qunit` depends on `ember-source`, which declared `webpack` as a peerDependency.
This makes sense for apps, but not for v2 addons.

The `webpack` peerdep in ember-source will also go away once it itself is converted to a v2 addon